apps/designer/app/designer/features/style-panel/sections/spacing/tooltip.tsx

```diff
@@ -0,0 +1,51 @@
+import { styled, Tooltip } from "@webstudio-is/design-system";
+import { SpacingStyleProperty } from "./types";
+
+// trigger is used only for positioning
+const Trigger = styled("div", {
+  position: "absolute",
+  width: "100%",
+  height: "100%",
+  pointerEvents: "none",
+  top: 0,
+  left: 0,
+});
+
+const labels = {
+  paddingTop: "Padding-top",
```

Comment:
We just had a discussion with @taylornowotny to use space separated upper cased names: `Padding Top`, he will update the designs next